src/pages/home/base/mobile-core/lifecycle/api-reference.md

```diff
@@ -82,9 +82,11 @@ iOS
 
 ## registerExtension
 
-<InlineAlert variant="warning" slots="text"/>
+<InlineAlert variant="warning" slots="header, text1"/>
 
-Deprecated as of 2.0.0. Please use the [MobileCore.registerExtensions](../api-reference.md#registerextensions) API instead.
+This API has been deprecated starting in v2.0.0 and removed in v3.0.0 of the Android extension.
+
+Use [`MobileCore.registerExtensions()`](../api-reference.md#registerextensions) API instead.
```

Comment:
maybe we could create even for this one a shared alert since it has the same message for all extensions

Reply:
It is not possible as each alert also includes some text regarding the alternate API to use. Let us follow with the DEVSITE team to see if there is a better way to capture and reuse stuff across pages.

Reply:
here I was referring to the registerExtension alert only which is the same across al extensions

Reply:
The relative path when inking to `MobileCore.registerExtensions` is different for base and other extensions. It is not possible to reuse a single one between all the extensions.

Reply:
ok, let's do it for the non-core extensions then